src/main/kotlin/com/infusionvlc/somniumserver/dreams/DreamController.kt

```diff
@@ -178,5 +181,11 @@ class DreamController(
       is DreamCreationErrors.DescriptionMissing -> "Description is missing"
       is DreamCreationErrors.InvalidDate -> "Invalid dreamt date"
       is DreamCreationErrors.CreatorNotFound -> "User with id ${error.userId} was not found"
+      is DreamCreationErrors.PersistenceError -> persistenceErrorString
```

Comment:
Why you extract this to a `val` and the others no? @Cotel

Reply:
Because it is being used in several places unlike the others.